### PR TITLE
[KBFS-1871] Make diskLimiter interface more descriptive

### PR DESCRIPTION
**Pull request by akalin-keybase**
Also move > 0 checking into implementation.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
Splitting this off from #690.

File: libkbfs/tlf_journal.go [hunk 1]
```diff
@@ -365,11 +365,11 @@ func makeTLFJournal(
 
 	// Do this only once we're sure we won't error.
 	storedBytes := j.blockJournal.getStoredBytes()
+	j.diskLimiter.onJournalEnable(storedBytes)
 	if storedBytes > 0 {
-		availableBytes := j.diskLimiter.ForceAcquire(storedBytes)
 		j.log.CDebugf(ctx,
-			"Force-acquired %d bytes for %s: available=%d",
-			storedBytes, tlfID, availableBytes)
+			"Force-acquired %d bytes for %s: %v",
```
Comment (strib) [inline libkbfs/tlf_journal.go]: My comment from the other PR applies here -- I feel like we should be printing out `storedBytes`, not `j.diskLimiter` here.  But I might be missing something.

Comment (strib) [inline libkbfs/tlf_journal.go]: Also, it's not really a "Force acquire" anymore.

File: libkbfs/tlf_journal.go [hunk 2]
```diff
@@ -1461,13 +1457,12 @@ func (j *tlfJournal) getBlockData(id kbfsblock.ID) (
...
 	timeout        time.Duration
 	requestedBytes int64
-	availableBytes int64
 	err            error
 }
 
 func (e ErrDiskLimitTimeout) Error() string {
-	return fmt.Sprintf("Disk limit timeout of %s reached; requested %d bytes, only %d bytes available: %+v",
-		e.timeout, e.requestedBytes, e.availableBytes, e.err)
+	return fmt.Sprintf("Disk limit timeout of %s reached; requested %d bytes: %+v",
```
Comment (strib) [inline libkbfs/tlf_journal.go]: It's a shame not to include the current disk usage here.  Will we be able to do that in a future PR once `getDiskLimits` is in?

File: libkbfs/tlf_journal.go (hunk 1 above)
Comment (akalin-keybase) [inline libkbfs/tlf_journal.go]: Oops, meant to fix that. Done, folded the bytes into an existing log.

File: libkbfs/tlf_journal.go (hunk 2 above)
Comment (akalin-keybase) [inline libkbfs/tlf_journal.go]: Ah, I put it back in. On an error, the available bytes may not be returned, but that can be accounted for.

**Pull request merged**